identity-actor/src/remote_account/mod.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+mod handler;
+mod requests;
+
+pub use handler::RemoteAccount;
+pub use requests::IdentityCreate;
+pub use requests::IdentityGet;
+pub use requests::IdentityList;
+
+#[derive(Debug, serde::Serialize, serde::Deserialize)]
+#[non_exhaustive]
+pub enum RemoteAccountError {
```

Comment:
Consider adding some documentation explaining that this is part of a PoC (which is why it is OK not to implement the Error trait here).